### PR TITLE
Issue #25 : Adding tests to ensure loading and saving params works

### PR DESCRIPTION
**Pull request by PonteIneptique**
This is a first step towards tests. It actually caught a bug with the saving and loading of nb epochs in tagger itself.

### CODE REVIEW

File: tests/test_configs.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        self.assertEqual(tagger.include_pos, True, "include_pos should be correctly loaded")
+        self.assertEqual(tagger.include_morph, False, "include_morph should be correctly loaded")
+        self.assertEqual(tagger.include_dev, True, "include_dev should be correctly loaded")
+        self.assertEqual(tagger.include_test, True, "include_test should be correctly loaded")
+        self.assertEqual(tagger.min_token_freq_emb, 5, "min_token_freq_emb should be correctly loaded")
+        self.assertEqual(tagger.halve_lr_at, 75, "halve_lr_at should be correctly loaded")
+        self.assertEqual(tagger.max_token_len, 20, "max_token_len should be correctly loaded")
+        self.assertEqual(tagger.min_lem_cnt, 1, "min_lem_cnt should be correctly loaded")
+
+    def test_load_after_save(self):
```
Comment (Jean-Baptiste-Camps) [inline tests/test_configs.py]: Why this redundancy ?

**Comment by Jean-Baptiste-Camps** [comment]
Great to have some tests now. What was the bug ?
Also, I am not 100% sure to get what overwrite does.

**Comment by PonteIneptique** [comment]
It's just a full test that allows for checking if everything stayed the
same or was correctly parsed.

**Comment by PonteIneptique** [comment]
Number epoch would be erased post load.

**Comment by Jean-Baptiste-Camps** [comment]
Ok, so overwrite is here if you want to provide a list of parameters to overwrite configuration ?

**Comment by PonteIneptique** [comment]
Overwrite has just been moved. It was already here.

**Comment by Jean-Baptiste-Camps** [comment]
Oups, my bad. That solved, I see no reason not to merge.

**Pull request merged**